bindings/nodejs/.prettierignore

```diff
@@ -3,3 +3,6 @@ generated.js
 generated.d.ts
 .yarn
 pnpm-lock.yaml
+.devbox
```

Comment:
Should not be included in this PR.

Reply:
https://github.com/apache/opendal/pull/4791 merged without those changes. Let's keep them as is to avoid extra works.

Reply:
sry, I forgot to move it to that PR